src/libraries/Microsoft.PowerFx.Interpreter/ReflectionFunction.cs

```diff
@@ -316,7 +332,15 @@ public async Task<FormulaValue> InvokeAsync(IServiceProvider serviceProvider, Fo
 
             foreach (var arg in args)
             {
-                args2.Add(arg);
+                if (arg is LambdaFormulaValue lambda)
+                {
+                    Func<Task<BooleanValue>> argLambda = async () => (BooleanValue)await lambda.EvalAsync(lambda.Runner, lambda.Context);
```

Comment:
>lambda.Runner, lambda.Context [](http://example.com/codeflow?start=106&length=29)

if we are passing them in here, do we still need them on the FormulaLambdaValue itself?